Payments/Operation/03_ManageRefunds.md

```diff
@@ -0,0 +1,77 @@
+[!!LIST (Refunds](../UserInterface/03_Refunds.md)
+
+
+# Monitor refunds
+
+A refund is a repayment of a capture. On the one hand, it is initialized by the trader, for example, if a product is not available and cannot be delivered. On the other hand, it is initialized if a payment and capture transaction has been manually refunded.
```

Comment:
In unserem System wird oft über "merchant" (für Händler) gesprochen.

" On the one hand, it is initialized by the trader, for example, if a product is not available and cannot be delivered. On the other hand, it is initialized if a payment and capture transaction has been manually refunded." -> Ist das nicht "das gleiche"? Oder besser gesagt, das erste (Produkt nicht verfügbar) ist der Grund/die Ursache für den zweiten Schritt (Refund anstoßen)? Also, it can be initialized by the merchant if, for example, a product is not available and cannot be delivered (or if customer has requested a refund for whatever reason). Then, the merchant initiates the refund process manually.